python/audit/utils.py

```diff
@@ -619,12 +634,17 @@ def _get_audit_operation_and_object(
                 object_name=config_group.object.name,
                 object_type=object_type,
             )
+
+            fqdn = None
             if res:
-                audit_operation.name = audit_operation.name.format(fqdn=res.data["fqdn"])
+                fqdn = res.data["fqdn"]
             elif "id" in view.request.data:
                 host = Host.objects.filter(pk=view.request.data["id"]).first()
                 if host:
-                    audit_operation.name = audit_operation.name.format(fqdn=host.fqdn)
+                    fqdn = host.fqdn
+
+            if fqdn:
+                audit_operation.name = audit_operation.name.format(fqdn=fqdn)
```

Comment:
I guess there'll still be "{fqdn}" in operation name, maybe change 
`name=f"{{fqdn}} host added to {config_group.name} configuration group"`
to
`name=f"host added to {config_group.name} configuration group"`
and then 
`audit_operation.name = f"{fqdn} {audit_operation.name}"`
?

Reply:
done